Source/WebKit/Platform/IPC/ArgumentCoders.h

```diff
@@ -674,26 +715,27 @@ template<typename ErrorType> struct ArgumentCoder<Expected<void, ErrorType>> {
 };
 
 template<typename... Types> struct ArgumentCoder<std::variant<Types...>> {
-    template<typename Encoder>
-    static void encode(Encoder& encoder, const std::variant<Types...>& variant)
+    template<typename Encoder, typename T>
+    static auto encode(Encoder& encoder, T&& variant)
+        -> std::enable_if_t<std::is_same_v<std::remove_cvref_t<T>, std::variant<Types...>>>
```

Comment:
An idiom we use elsewhere is to add a template parameter of std::enable_if_t<something>* = nullptr instead of using it as a return type.  It makes it more straightforward to see what the return type is.

Reply:
OK, I think that should be doable everywhere.